### PR TITLE
Introduce multi host in pulsar doc

### PR DESCRIPTION
**Pull request by zymap**
### Motivation

Pulsar already have been supported multi host. Introduce them in Pulsar document.

### CODE REVIEW

**Comment by zymap** [comment]
@sijie @Anonymitaet @Jennifer88huang PTAL.

**Comment by Jennifer88huang** [comment]
@Anonymitaet Could you please help review this file? Thank you.

**Comment by Anonymitaet** [comment]
@zymap @Jennifer88huang Glad to help.

File: site2/docs/admin-api-overview.md [hunk 1]
```diff
@@ -70,3 +70,23 @@ config.setTlsTrustCertsFilePath(tlsTrustCertsFilePath);
 
 PulsarAdmin admin = new PulsarAdmin(url, config);
 ```
+
+If you have mutilple brokers to using, you can use multi host like pulsar service, like this:
```
Comment (Anonymitaet) [inline site2/docs/admin-api-overview.md]:
```suggestion
If you have multiple brokers to use, you can use multi-host like Pulsar service. For example,
```

File: site2/docs/admin-api-overview.md [hunk 2]
```diff
@@ -70,3 +70,23 @@ config.setTlsTrustCertsFilePath(tlsTrustCertsFilePath);
...
+```java
+URL url = new URL("http://localhost:8080,localhost:8081,localhost:8082");
+// Pass auth-plugin class fully-qualified name if Pulsar-security enabled
```
Comment (Anonymitaet) [inline site2/docs/admin-api-overview.md]:
```suggestion
// Pass auth-plugin class fully-qualified name if Pulsar-security is enabled.
```

Comment (Anonymitaet) [reply site2/docs/admin-api-overview.md]: Are "auth-plugin" and "Pulsar-security" something like a prop noun?

File: site2/docs/client-libraries-java.md [hunk 3]
```diff
@@ -57,6 +57,11 @@ Pulsar protocol URLs are assigned to specific clusters, use the `pulsar` scheme
 pulsar://localhost:6650
 ```
 
+If you have more than one broker, the URL will look like something like this:
```
Comment (Anonymitaet) [inline site2/docs/client-libraries-java.md]:
```suggestion
If you have more than one broker, the URL may look like this:
```

File: site2/docs/client-libraries-java.md [hunk 4]
```diff
@@ -79,6 +84,13 @@ PulsarClient client = PulsarClient.builder()
         .build();
 ```
 
+If you have mutliple brokers, you can instantiate a PulsarClient like this:
```
Comment (Anonymitaet) [inline site2/docs/client-libraries-java.md]:
```suggestion
If you have multiple brokers, you can initiate a PulsarClient like this:
```

File: site2/docs/deploy-bare-metal.md [hunk 5]
```diff
@@ -43,6 +43,8 @@ To run Pulsar on bare metal, you are recommended to have:
 > However if you don't have enough machines, or are trying out Pulsar in cluster mode (and expand the cluster later),
 > you can even deploy Pulsar in one node, where it will run zookeeper, bookie and broker in same machine.
 
+> If you don't have a DNS server, you can use multi host in service url to replacing the DNS.
```
Comment (Anonymitaet) [inline site2/docs/deploy-bare-metal.md]:
```suggestion
> If you don't have a DNS server, you can use multi-host in service URL instead.
```

File: site2/docs/admin-api-overview.md (hunk 2 above)
Comment (zymap) [reply site2/docs/admin-api-overview.md]: I think not.

**Comment by zymap** [comment]
@Anonymitaet Thanks for your review.

**Comment by zymap** [comment]
run java8 tests

**Comment by zymap** [comment]
run java8 tests

**Pull request merged**